rust_snuba/rust_arroyo/src/backends/kafka/config.rs

```diff
@@ -23,6 +24,7 @@ impl KafkaConfig {
         auto_offset_reset: String,
         _strict_offset_reset: bool, // TODO: Implement this
         override_params: Option<HashMap<String, String>>,
+        consumer_config_params: Option<HashMap<String, String>>,
```

Comment:
This argument should not be necessary - there is already a way to pass consumer consumer config via `override_params` in the argument above this